doc/explanation/lxc_show_info.md

```diff
@@ -0,0 +1,50 @@
+(lxc-show-info)=
+# About `lxc` `show` and `info`
+
+For the entities managed by LXD, the `lxc` command provides a 'list' sub-command, and might provide `show` and `info` sub-commands.
+The purpose of the `info` sub-command is to show current state information, and the purpose of the `show` sub-command is to show configuration information and how the entity is used by other entities.
+
+For example, the `lxc network info` command shows IP address and traffic statistics:
+
+    Name: lxdbr0
+    MAC address: 00:16:3e:d3:ec:41
+    MTU: 1500
+    State: up
+
+    Ips:
+      inet    10.247.40.1
+      inet6   fd42:d389:f4a1:53d2::1
+      inet6   fe80::216:3eff:fed3:ec41
```

Comment:
This is a link-local address so I think it can stay as it is (I'm confirming with the devs).